React/Modules/RCTDevMenu.m

```diff
@@ -228,224 +178,45 @@ - (instancetype)init
     [commands registerKeyCommandWithInput:@"i"
                             modifierFlags:UIKeyModifierCommand
                                    action:^(__unused UIKeyCommand *command) {
-                                     [weakSelf.bridge.eventDispatcher
-#pragma clang diagnostic push
-#pragma clang diagnostic ignored "-Wdeprecated-declarations"
-                                      sendDeviceEventWithName:@"toggleElementInspector"
-                                      body:nil];
-#pragma clang diagnostic pop
+                                     [weakSelf.bridge.devSettings toggleElementInspector];
                                    }];
 
     // Reload in normal mode
     [commands registerKeyCommandWithInput:@"n"
                             modifierFlags:UIKeyModifierCommand
                                    action:^(__unused UIKeyCommand *command) {
-                                     weakSelf.executorClass = Nil;
+                                     [weakSelf.bridge.devSettings updateSettingWithValue:@(NO) forKey:kRCTDevSettingIsDebuggingRemotely];
```

Comment:
I'd prefer a stronger interface on `RCTDevSettings` rather than `updateSettingWithValue:forKey`

nit: use `@NO` instead of `@(NO)`